mkdocs.yml

```diff
@@ -26,16 +26,16 @@ pages:
     - "Create your first app": dev/app.md
     - "Develop a connector": dev/konnector.md
     - "Create a mobile app with cordova": dev/cordova.md
-theme: readthedocs
-theme_dir: 'cozy-theme'
+theme: mkdocs
+# theme_dir: 'cozy-theme'
```

Comment:
By commenting out this line, you remove the customization from this folder:
 - we have overridden the default template to add a navigation link between english and french language, and to add Piwik;
 - this folder also contains the 404 page.

So, what we should probably do is copy the `base.html` file from the `mkdocs` theme into this folder and update it to add piwik and so on;